.github/workflows/tests.yml

```
@@ -0,0 +1,28 @@
+name: Unit tests
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ["20", "18", "16"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
```

```suggestion
          node-version: ${{ matrix.node-version }}
          cache: 'npm'
```

cache npm to speedup